src/features/hud/components/InventoryItems.tsx

```diff
@@ -4,35 +4,154 @@ import { OuterPanel, Panel } from "components/ui/Panel";
 import { Context } from "features/game/GameProvider";
 import { ITEM_DETAILS } from "features/game/types/images";
 import { InventoryItemName } from "features/game/types/game";
-import React, { useContext, useState } from "react";
+import { KNOWN_IDS, CATEGORY_ID, CATEGORY } from "features/game/types";
+import React, { useContext, useState, useRef } from "react";
+import { Tab } from "components/ui/Tab";
 
-export const InventoryItems: React.FC = () => {
+import close from "assets/icons/close.png";
+import arrowLeft from "assets/icons/arrow_left.png";
+import arrowRight from "assets/icons/arrow_right.png";
+import seed from "assets/crops/beetroot/seed.png";
+import crop from "assets/crops/sunflower/crop.png";
+import tool from "assets/tools/hammer.png";
+import nft from "assets/nfts/gnome.png";
+import food from "assets/crops/wheat/flour.png";
+import resource from "assets/resources/wood.png";
+interface Props {
+  onClose: () => void;
+}
+
+export const categories: CATEGORY[] = [
```

Comment:
Instead of declaring a separate category ID type (which is directly typed to the items), consider the following pattern.


```
type Tab = 'Seeds' | 'Crops' | ...
export const CATEGORIES: Record<Tab, { img: any, items: object }  = {
  Seeds: {
   img: seed,
   // import the items
   items: SEEDS
},  Crops': {
   img: crop,
   // import the items
   items: CROPS
},
...
}
...
```